src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -524,10 +524,10 @@ private void PopulateProtocol()
         internal static string GetLocalDBInstance(string dataSource, out bool error)
         {
             string instanceName = null;
-            // ReadOnlySpan is not supported in netstandard 2.0, but installing System.Memory solves the issue
             ReadOnlySpan<char> input = dataSource.AsSpan().TrimStart();
             error = false;
             // NetStandard 2.0 does not support passing a string to ReadOnlySpan<char>
```

Comment:
@JRahnama Unsure how this can be improved now?

Reply:
Well, I think we can remove System.Memory and check the exception maybe?

Reply:
Sorry, I have no idea what you are referring to...

Reply:
System.memory is no longer used - are you suggesting I just remove the second comment also?